projects/utils-browser/src/public-api.ts

```diff
@@ -4,4 +4,3 @@
 
 export * from './lib/download/download';
 export * from './lib/layout/layout';
-export * from './lib/layout/colors';
```

Comment:
How could you remove this file without including a library like chroma for it afterwards becasue I don't see any change in the package.json, or did you don't use the functions before?

Reply:
Those functions were only used on the client side - in another program depending on the ukis-frontend-libraries. 
That program has now been updated to use chroma instead.